source/tests/run_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+help(){
+  cat <<EOF
+    Usage: $0 [arguments]
+    Available options:
+    -h, --help      Print this help and exit.
+    -rut, --run_unit_test    Run Unit Test.
+EOF
+	exit 0
+}
+
+if [[ ( $@ == "--help") ||  $@ == "-h" ]]
+then
+    help
+fi
+
+if [[ ( $@ == "--run_unit_test") ||  $@ == "-rut" ]]
+then
+
+    VENV=$(mktemp -d) && echo "$VENV"
+    python3.10 -m venv "$VENV"
+    source "$VENV"/bin/activate
+
+    export AMC_ENDPOINT_URL="https://test_end_point_url.com/test"
+    export AMC_API_ROLE_ARN="arn:aws:iam::999999999999:role/SomeTestRole"
+    export SOLUTION_NAME="SOLUTION_NAME"
+    export SOLUTION_VERSION="SOLUTION_VERSION"
+    export VERSION="0.0.1"
+    export botoConfig='{"region_name": "us-east-1"}'
+    export AWS_XRAY_SDK_ENABLED=false
+    export AMC_GLUE_JOB_NAME="some-GlueStack-123-amc-transformation-job"
+
+    pip install -r requirements-dev.txt
+    pytest tests -vv --ignore="tests/e2e/"
```

Comment:
All the unit tests should be in source/tests/unit/ rather than under /source/tests/ for organizational purposes. Then it will also not be necessary to use --ignore to avoid running e2e tests from the run_unit.sh script.

Eventually we'll want to move integ tests from source/api/test/ to /source/tests/integ/. See https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/issues/106

Reply:
yes i tried to do this, but chalice imports doesn't work very well outside `source/app.` Probably have to do some hacks to make the imports available outside of `source/app`. This is the reason why we have two `sigv4.py`.  SonarQube still complain about this but its low priority

Reply:
in the `.run_unit_test.py` i just combined the coverage report files using` pytest --cov-append`.

Reply:
If it would be time consuming to move unit tests to source/tests/unit/, I think it would be justified to include that task under #106.

Reply:
Yes, i will also update that issue to include using only one sigv4.py file. To solve duplicate code issue in SonarQube.

Reply:
#107